Rivet/Functions/Convert-FileInfoToMigration.ps1

```diff
@@ -34,63 +34,63 @@ function Convert-FileInfoToMigration
 
     process
     {
-        $InputObject | 
-            ForEach-Object {
-                $dbName = Split-Path -Parent -Path $_.FullName
-                $dbName = Split-Path -Parent -Path $dbName
-                $dbName = Split-Path -Leaf -Path $dbName
-
-                $m = New-Object 'Rivet.Migration' $_.MigrationID,$_.MigrationName,$_.FullName,$dbName
-                Write-Timing -Message ('Convert-FileInfoToMigration  {0}' -f $m.FullName)
-
-                filter Add-Operation
-                {
-                    param(
-                        [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
-                        [object]
-                        # The migration object to invoke.
-                        $Operation,
-
-                        [Parameter(ParameterSetName='Push',Mandatory=$true)]
-                        [Collections.Generic.List[Rivet.Operations.Operation]]
-                        [AllowEmptyCollection()]
-                        $OperationsList,
+        filter Add-Operation

```

Comment:
`filter`? That's a new one to me >_>

Reply:
It's shorthand for functions that take input in a pipeline:

```
function Name
{
    begin
    {
        # Called before first item is piped to function.
    }
    process
    {
        # Called for each item piped to the function.
    }
    end
    {
        # Called after last item is piped to function
    }
}
```

Or, with a filter:
```
function Name
{
    # Called for each item piped to the function.
}
```